packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -754,6 +754,27 @@ class VotesRepo extends AbstractRepo<"Votes"> {
     }
     return result;
   }
+  async getNetKarmaChangesForAuthorsOverPeriod(days: number, limit: number): Promise<Array<{ userId: string; netKarma: number }>> {
+    return this.getRawDb().any(`
```

Comment:
This query is kind of expensive (having run an EXPLAIN ANALYZE on it) but _probably_ not expensive enough to block launch, as long as it's not on the home page.